src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/CaseMetadataControllerTest.java

```diff
@@ -156,6 +173,8 @@ private static void assertValid(Claim dto, CaseMetadata metadata) {
         assertEquals(dto.getClaimantRespondedAt().orElse(null), metadata.getClaimantRespondedAt());
         assertEquals(dto.getSettlementReachedAt(), metadata.getSettlementReachedAt());
         assertEquals(dto.getClaimDocument(SEALED_CLAIM), Optional.ofNullable(metadata.getSealedClaimDocument()));
-        assertEquals(dto.getClaimData().getPayment().getReference(), metadata.getPaymentReference());
+        if (dto.getClaimData().getPayment() != null) {
+            assertEquals(dto.getClaimData().getPayment().getReference(), metadata.getPaymentReference());
+        }
```

Comment:
else you may want to check `assertNull(metadata.getPaymentReference())`

Reply:
Agreed